test/extended/util/db/mongodb.go

```diff
@@ -0,0 +1,62 @@
+package db
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/origin/test/extended/util"
+)
+
+// MongoDB is a MongoDB helper for executing commands.
+type MongoDB struct {
+	PodName       string
+	MasterPodName string
+}
+
+// NewMongoDB queries OpenShift for a pod with given name, saving environment
+// variables like username and password for easier use.
+func NewMongoDB(podName, masterPodName string) util.Database {
+	if masterPodName == "" {
+		masterPodName = podName
+	}
+	return &MongoDB{
+		PodName:       podName,
+		MasterPodName: masterPodName,
+	}
+}
+
+func (m MongoDB) GetPodName() string {
+	return m.PodName
```

Comment:
`Database` is an interface type, and the method is part of that interface: https://github.com/openshift/origin/pull/6197/files#diff-7696fba4f9bfabdb09dbc5a43c0223e4R15. Perhaps we can make it better by just stating the purpose of the method:

```go
// GetPodName implements Database.
func (m MongoDB) GetPodName() string {
	return m.PodName
}
```

In that case the field name could be indeed unexported. In Go we normally don't use "Get" in getter names: https://golang.org/doc/effective_go.html#Getters.